components/button/test/button.visual-diff.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../colors/colors.js';
```

Comment:
I think this was removed from the other visual-diff tests.  Is it needed here?  I'm curious where colors is being pulled in from, in both the demos, and visual-diff cases.

Reply:
oh, may bad I thought I'd gotten rid of those.